solver/llbsolver/file/refmanager.go

```diff
@@ -6,17 +6,19 @@ import (
 	"github.com/moby/buildkit/cache"
 	"github.com/moby/buildkit/session"
 	"github.com/moby/buildkit/snapshot"
+	"github.com/moby/buildkit/solver"
 	"github.com/moby/buildkit/solver/llbsolver/ops/fileoptypes"
 	"github.com/moby/buildkit/util/bklog"
 	"github.com/pkg/errors"
 )
 
-func NewRefManager(cm cache.Manager) *RefManager {
-	return &RefManager{cm: cm}
+func NewRefManager(cm cache.Manager, vtx solver.Vertex) *RefManager {
```

Comment:
Better to avoid importing whole `solver.Vertex` in here. Let's just add a name parameter to here.

Reply:
Sure.

for `RefManager`, this should be as `custom desc`, so I name the parameter `desc` instead.